pkg/apiserver/api_installer.go

```diff
@@ -812,12 +812,14 @@ func typeToJSON(typeName string) string {
 		return "boolean"
 	case "uint8", "int", "int32", "int64", "uint32", "uint64":
 		return "integer"
-	case "byte":
-		return "string"
 	case "float64", "float32":
 		return "number"
+	case "byte":
+		return "string"
 	case "util.Time":
 		return "string"
+	case "[]string":
```

Comment:
I am trying to figure out a way to express this as 
```
"type": "array",
      "items": {
       "type": "string"
      },
```
https://github.com/emicklei/go-restful/issues/225

Will update the PR once I have it working